bp-activity-subscription-functions.php

```diff
@@ -1782,3 +1782,13 @@ function ass_weekly_digest_week() {
 	elseif ( $ass_weekly_digest == 0 )
 		return __('Sunday' );
 }
+
+/**
+ * Register our theme template directory with BuddyPress.
+ */
+function bpges_register_template_stack() {
+	bp_register_template_stack( function() {
```

Comment:
We shouldn't register the template stack until we're on our actual page.

Registering template stacks adds to the filesystem IO checks.

Reply:
Ah, hadn't thought of that. Only caveat is that if we decide to add more theme compat somewhere (not sure where this would be) we would need to remember to also loosen the conditional.